### DISCUSSION TITLE
v1.11.0 - External Web Client

### DISCUSSION THREAD

**Discussion by iwalton3** [jellyfin/jellyfin-desktop#682] [General]
This release switches Jellyfin Media Player to always use the external web client so that updates to `jellyfin-web` take effect immediately instead of when I get a chance to release the application.

About Windows Downloads:
 - The desktop installer requires administrator rights.
 - You can avoid warnings on files by unblocking the file in the properties dialog.
 - You can manually build a release [using these instructions](https://github.com/jellyfin/jellyfin-media-player#building-for-windows).

<hr /><em>This discussion was created from the release <a href='https://github.com/jellyfin/jellyfin-media-player/releases/tag/v1.11.0'>v1.11.0 - External Web Client</a>.</em>

**Comment by TheSpyder** [comment]
I'm on macOS Apple Silicon and this version seems to not read my previous settings so it was asking for the server details. I initially downgraded back to v1.10.1 (which loaded up immediately) because I couldn't get 1.11 to connect to my server at all, but specifying the 8096 port finally got it working.

Server version 10.9.2.

**Reply by iwalton3** [reply]
Yup this version you have to log in again due to the change in how it loads the web client.

**Reply by Tirpitz** [reply]
Same problem. The client doesn't seem to find/connect to the server on macOS Silicon.

**Comment by vincejv** [comment]
No longer connects to my server after the Client upgrade, using reverse proxy subpath/subfolder
Server: Jellyfin version: 10.9.5 also doesn't work with 10.9.6
Installed through flathub
OS: Kubuntu 22.04 LTS
Not having issues with other clients, web (chrome), android and iOS..

can't even provide any logs as it seems like it doesn't even reach the server at all, as no logs are produced on the server on the connection attempt

**Reply by vincejv** [reply]
![image](https://github.com/jellyfin/jellyfin-media-player/assets/1276544/3ecbc49a-a0a6-4944-bffa-150416888e4c)

**Reply by vincejv** [reply]
well apparently people raised the issues already

https://github.com/jellyfin/jellyfin-media-player/issues/683
https://github.com/jellyfin/jellyfin-media-player/issues/684

**Comment by DigitalCharon** [comment]
Same issue. Cannot connect to my server through either specifying the 8096 port or through the port 80 proxy I have setup. Specifying the port hasn't worked for awhile but now I can't even get around that.

**Comment by birdup420** [comment]
Sorry if this is a dumb question but does this mean the client's interface will always actually be running on the server? Or if the client has an up to date jellyfin-web it will run its own? Will this increase the load on the server (even if little)? Thanks for the amazing work you're all doing btw

**Reply by iwalton3** [reply]
The client will load the web client from the server. It doesn't ship its own anymore allowing for the one which is kept up to date by the server to be used. It will lightly increase server load, but it is unlikely to be significant.

**Reply by birdup420** [reply]
Thanks for the reply! But even if the load is minimal, why not change the default setting instead of removing it completely? I'm trying to run my server with as little cpu load as possible, so I preferred the clients running their own web client even if it was a little out of date

**Comment by wnhre2ur8cxx8** [comment]
So using Jellyfin Media Player without running jellyfin-web on the server is not possible anymore? Or do I understand something wrong?

**Reply by wnhre2ur8cxx8** [reply]
Why such a big change? :scream: what if I dont want to have a full web server running?

**Reply by thornbill** [reply]
It's virtually impossible to run Jellyfin without the web interface already as there is no other practical way to setup a server and many clients already require the web interface to function.

**Comment by zhijiee** [comment]
On Silicon mac, this update broke the rendering of ASS subtitle. Not sure if i am doing something wrongly.

**Reply by BearPawGames** [reply]
Subtitles in general are all messed up after this update for me. The subtitles only render in the lower left corner, and each letter of the sentence is stacked on top of one another. I found that if I minimize the player then maximize it again the subs will display correctly.
